lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3921,6 +3921,28 @@ def model_find_and_add_construction(model, climate_zone_set, intended_surface_ty
                                 'building_category' => building_category,
                                 'minimum_percent_of_surface' => minimum_percent_of_surface,
                                 'maximum_percent_of_surface' => maximum_percent_of_surface)
+
+      # 12/14/2021 added to test data pull from sub climate zones for transparent surfaces
+      if !props
+        if climate_zone_set == "ClimateZone 3"
```

Comment:
Looks like there's not `else` so these could probably be on the same line.

Reply:
Also, I'm not sure if this needs to be specific to climate zone 3. There might be other climate zone sets having similar issues. What do you think?

Reply:
Possibly. For now it only addresses for Climate Zone 3 because 2004/PRM requirements. The future development could add more similar cases.